frontend/src/components/searchBar/GameChangerSearchBar.js

```diff
@@ -474,15 +474,15 @@ const GameChangerSearchBar = (props) => {
 					</GCTooltip>
 				}
 				{ dropdownOpen && !advancedSearchOpen && <SearchBarDropdown searchText={searchText} rowData={dataRows} cursor={cursor} /> }
-				{ isGamechanger && hideSearchResults &&
+				{ (isGamechanger || isEDA) && hideSearchResults &&
```

Comment:
This needs to be moved into a module and not in the main component. @ryanryanchan @aaron42051 Can both you work this together as it may have been added by Ryan previously and I didn't catch it then.